src/main/resources/avro/referencemethods.avdl

```diff
@@ -113,9 +113,9 @@ as JSON.
 */
 record SearchReferencesRequest {
   /**
-  If present, return only references which belong to this reference set.
+  The ReferenceSet to search.
   */
-  union { null, string } referenceSetId = null;
```

Comment:
@calbach, see https://github.com/ga4gh/schemas/issues/247 regarding where this is stemming from.  Though I agree that many-to-many is something that I think is still important, it probably requires more clarity with examples in the structure of the definitions.  Just for reference for others, the original many-to-many was discussed initially in the following issue:

https://github.com/ga4gh/schemas/issues/125#issuecomment-54029648

You can still search with MD5 via this in the same method:

```
  /**
  If nonempty, return references which match any of the given `md5checksums`.
  See `Reference::md5checksum` for details.
  */
  array<string> md5checksums = [];

```

~p